UI/dash.html

```diff
@@ -0,0 +1,131 @@
+                                </div>
+                                <div>
+                                    <input type="radio" id="self-delivery" name="self-delivery" value="self-delivery">
+                                    <label for="self-delivery">pickup</label>
+                                    <input type="radio" id="delivered" name="delivered" value="delivered">
+                                    <label for="delivered">delivery</label>
+                                </div>
+                                <div>
+                                    <input type="submit" name="submit"  value="buy" id="submit" class="button-sunshine-outline-order">
+                                </div>
+                            </form>    
+                        </div>
+                    </div>
+            
+                </div>
+            </div>
+
+<!-- menu items displaying images -->
+
+            <div style="cursor:pointer;" class="menu_item">
+                <div class="snack-data">
+                    <small>STREET BURGERS</small>
+                </div>
+                <h2 class="snack-title">El Chapo</h2>
+                <img src="img/burger.jpg" >
+                <p class="snack-content">Slow Grilled beef patty with a slice of cheddar cheese,
+                    caramelised onions, tomatoes, ketchup and mayo</p>
+            </div>
+            <div style="cursor:pointer;" class="menu_item">
+                <div class="snack-data">
+                    <small>STREET BURGERS</small>
+                </div>
+                <h2 class="snack-title">Double El Chapo</h2>
+                <img src="img/double.PNG">
+                <p class="snack-content">2 Slow Grilled beef patty with double slice of cheddar cheese,
+                    caramelised onions, tomatoes, ketchup and mayo</p>
+            </div>
+            <div style="cursor:pointer;" class="menu_item">
+                <div class="snack-data">
+                    <small>WRAPS</small>
+                </div>
+                <h2 class="snack-title">SilverBack Wrap</h2>
+                <img src="img/silver.JPG">
+                <p class="snack-content">Shredded chicken marinated in a fusion 
+                    of spices topped with a hand full of fries, 
+                        guacomole, ketchup and mayonnaise wrapped in a tortilla wrap</p>
+            </div>
+            <div style="cursor:pointer;" class="menu_item">
+                <div class="snack-data">
+                    <small>KUNGPAO CHICKEN</small>
+                </div>
+                <h2 class="snack-title">2pc Kungpao</h2>
+                <img src="img/kungpao.JPG">
+                <p class="snack-content">2pc deep fried kungpao chicken with 
+                    a filling pack of fries and free drink</p>
+            </div>
+            <div style="cursor:pointer;" class="menu_item">
```

Comment:
good work but i would suggest you try to keep all your style within the .css files

Reply:
yea am with @kyakusahmed keep it in the external css... if they are bigger project there might be a possibility of confusion... the rest is really great